src/Cardano/Network/Name.hs

```diff
@@ -0,0 +1,16 @@
+module Cardano.Network.Name
+  ( NetworkName (..)
+  ) where
```

Comment:
There's no need for an extra module here, let's put that in the `HTTPBridge.Client` for now.

Reply:
I've moved this into `HttpBridge.Api` (as it's needed in both `Api` and `Client`).